harness/determined/common/experimental/determined.py

```diff
@@ -582,3 +582,21 @@ def stream_trials_validation_metrics(
             stacklevel=2,
         )
         return trial._stream_validation_metrics(self._session, trial_ids)
+
+    def list_tokens(self, includeInactive: Optional[bool] = None) -> List[user.TokenInfo]:
+        def get_with_offset(offset: int) -> bindings.v1GetAllAccessTokensResponse:
+            return bindings.get_GetAllAccessTokens(
+                session=self._session,
+                offset=offset,
+                includeInactive=includeInactive,
+            )
+
+        resps = api.read_paginated(get_with_offset)
+        userSessions = []
```

Comment:
Please also avoid camel casing in our code.  This isn't as critical as it isn't a user API.

I suppose I should also mention that the generated bindings have weird casing that isn't pythonic because generated bindings are just worse than hand-written code, at least from an ergonomic perspective.  But prefer snake_case in the python code that you hand write.

Reply:
Also, why is this variable called "user sessions" instead of like, "user tokens" or something?